buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UiTests.kt

```diff
@@ -28,11 +28,10 @@ fun Project.configureUiTests() {
       testClassesDirs = uiTest.output.classesDirs
       classpath = configurations["uiTestRuntimeClasspath"] + uiTest.output
 
-      dependsOn(":buildPlugin")
       val buildPlugin = tasks.findByPath(":buildPlugin")!!
-
-      systemProperty("ui-test.intellij.version", version)
```

Comment:
System properties could as well be used here... but Typesafe config doesn't support them for some reason